### ISSUE TITLE
Support bigint in queries

### ISSUE THREAD

**Issue by darg-2036** [G4brym/workers-qb#44]
Would it be possible to support `bigint` in this library? I get errors like the following when I try to use them:

```
  Overload 2 of 2, '(params: UpdateWithoutReturning): Query<D1Result>', gave the following error.
    Type 'bigint' is not assignable to type 'string | number | boolean | Raw | null'.
```

Thanks!

**Comment by G4brym** [comment]
Hey @darg-2036 i've just released a new version (v1.2.4) can you check if this solves your issue?
Thanks

**Comment by darg-2036** [comment]
I believe it does. Thanks!
